transports/openai-realtime-webrtc-transport/src/OpenAIRealTimeWebRTCTransport.ts

```diff
@@ -143,6 +143,8 @@ export class OpenAIRealTimeWebRTCTransport extends Transport {
       this._attachDeviceListeners();
     }
 
+    this.initDevices();
```

Comment:
i don't think we want this, but i'm curious the impetus for adding it. this `initialize()` gets called from the `RTVIClient`'s constructor and feels like a weird action to turn on your devices. Right now, we've been working on the theory that if a user wants to initialize devices before `connect()` they can manually call `initDevices()`, otherwise, they automatically turn on/init on `connect()`.